AT_CSC/ATDome/docker-compose.yaml

```diff
@@ -3,8 +3,8 @@ version: '3.6'
 services:
 
     atdome-csc:
-        image: lsstts/at_dome_sim:3.9-0.5
-        container_name: atdome-sim
+        image: lsstts/at_dome:3.9-0.5
+        container_name: atdome

```

Comment:
Do we want the actual done on the develop  branch?    It seems like we need a compose for the mountain and another for Tucson ..

Reply:
Not sure I understood the question. But yes, we will need different compose files for Tucson and for the Summit.